server/models.py

```diff
@@ -0,0 +1,72 @@
+from app import db
+from sqlalchemy.sql import func
+from sqlalchemy import ForeignKey
+
+class contest_user(db.Model):
+    __tablename__ = 'contest_user'
+
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
id is built-in function name for python. 

Let's use "username" for consistency.

Reply:
Id is used as a reference in other tables, that's why I used built-in function(when row creates, id is Automatically assigned to row) so in future conflict doesn't happen.

Reply:
I see what you mean. 
Then we should also include username as a col since that's the only way users are able to identify themselves during login.

Reply:
that's a good idea. I am just gonna apply.